pkg/cli/cvd.go

```diff
@@ -159,6 +162,50 @@ func (c *cvdCreator) Create() ([]*hoapi.CVD, error) {
 	return c.createCVDFromAndroidCI()
 }
 
+const uaEnvConfigTmpl = `
+{
+  "common": {
+    "host_package": "@user_artifacts/{{.ArtifactsDir}}"
+  },
+  "instances": [
+    {
+      "vm": {
+        "memory_mb": 8192,
+        "setupwizard_mode": "OPTIONAL",
+        "cpus": 8
+      },
+      "disk": {
+        "default_build": "@user_artifacts/{{.ArtifactsDir}}"
+      },
+      "streaming": {
+        "device_id": "cvd-1"
+      }
+    }
+  ]
+}
+`
+
+type uaEnvConfigTmplData struct {
+	ArtifactsDir string
+}
+
+func (d *uaEnvConfigTmplData) Value() map[string]interface{} {
+	tmpl, err := template.New("").Parse(uaEnvConfigTmpl)
+	if err != nil {
+		panic(err)
```

Comment:
Why does this function use `panic` to handle error, though all callers of this function already does error propagation?

Reply:
my thinking was that an error parsing the template should not be propagated, however the other two errors in this method should be propagated. I moved parsing the template into an `init` block.